compiler/optimizations/remoteValueForwarding.cpp

```diff
@@ -84,6 +88,83 @@ void remoteValueForwarding() {
   }
 }
 
+static void fixLHSArray(CallExpr* move, std::vector<Symbol*>& todo) {
```

Comment:
I'd rename this function since it applies to any record-wrapped type. If replaceRecordWrappedRefs were in its own file, it could juts be called fixLHS...